core/modules/system/includes/block.title_components.inc

```diff
@@ -0,0 +1,138 @@
+<?php
+
+/**
+ * TitleComponents extends Block
+ *
+ * This class allows us to display the page title, local task tabs, 
+ * local actions links, and messages in a block.
+ */
+class TitleComponents extends Block {
```

Comment:
Just as a name, how do you feel about renaming `TitleComponents` into something like `PageComponents`? I don't think of things like tabs or messages to be strictly part of the "title".